__factories__/clients/github.js

```diff
@@ -45,3 +45,17 @@ export const githubProfileFactory = new Factory().attrs({
     private_repos: 10000,
   },
 });
+
+export const githubRepositoryFactory = new Factory().attrs({
```

Comment:
This looks like it’s in the shape of our own stored Firebase data, not something we would get back from the GitHub client, right? If that’s correct I would put it in a separate top-level factory module, along the lines of `__factories__/data/firebase.js` (I don’t think it belongs in `clients/firebase.js` because the shape is determined by how we choose to store the data, not anything intrinsic about the Firebase client).